src/plugins/csp_configuration_provider/opensearch_dashboards.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "cspConfigurationProvider",
+  "version": "1.0.0",
+  "opensearchDashboardsVersion": "opensearchDashboards",
+  "server": true,
+  "ui": false,
+  "requiredPlugins": ["navigation"],
```

Comment:
does this need navigation plugin?

Reply:
Actually not. It was automatically added by the plugin creation command. Let me remove it.